agent-control/src/agent_control/config.rs

```diff
@@ -181,7 +181,7 @@ impl<'de> Deserialize<'de> for OpAMPClientConfig {
             #[serde(default)]
             fleet_id: String,
             #[serde(default)]
-            pub signature_validation: SignatureValidatorConfig,
+            pub signature_validation: Option<SignatureValidatorConfig>,
```

Comment:
I changed the interface to disable the validation in case no config is added but realized that makes more dificult to manage overrides of this in the chart. I will revert back to the `enabled` flag approach, it's also more extensible